examples/llm_qlora_data_parallel/README.md

```diff
@@ -0,0 +1,28 @@
+# Data-Parallel QLoRA Fine-Tuning
+
+If you have a single-node multi-GPU setup with a large dataset that you would like to train using QLoRA, you can use DeepSpeed Stage 0, 1, or 2.
+
+As a refresher, here is what each DeepSpeed Zero stage corresponds to:
+
+- **Stage 0**: Disabled, i.e., no partitioning of optimizer state, gradients or model parameters. You can still perform optimizer and parameter offloading, as well training using bf16 or fp16 etc.
+- **Stage 1**: The optimizer states (e.g., for Adam optimizer, 32-bit weights, and the first, and second moment estimates) are partitioned across the processes, so that each process updates only its partition.
+- **Stage 2**: The reduced 32-bit gradients for updating the model weights are also partitioned such that each process retains only the gradients corresponding to its portion of the optimizer states.
+- **Stage 3**: The 16-bit model parameters are partitioned across the processes. ZeRO-3 will automatically collect and partition them during the forward and backward passes.
+
+_NOTE: Data Parallel QLoRA based training only works with DeepSpeed stages \<= 2. This is because DeepSpeed isn't
```

Comment:
What is `\<=`?

Reply:
I believe it's automatically added to escape the italics formatted, here's what it looks like when rendered:

<img width="1291" alt="Screenshot 2023-10-19 at 1 41 34 PM" src="https://github.com/ludwig-ai/ludwig/assets/106701836/4849b6b1-f279-4b3b-a07d-ce8b22b32c70">